### DISCUSSION TITLE
Can HD textures be used compressed?

### DISCUSSION THREAD

**Discussion by solomongrundy6** [PCSX2/pcsx2#11554] [General]
Can the HD texture replacement feature access textures that are compressed, in a ZIP or whatever?  They're very large and compress a lot, the ability to use them compressed (or even decompress them somewhere temporarily on load) could save even upwards of a hundred gigabytes of space with a large collection of HD textures.

**Comment by F0bes** [comment]
We support PNG and DDS texture replacement formats.
DDS supports different compression modes.

NTFS (Windows) has transparent filesystem compression.
BTRFS (not Ext4) on Linux has transparent filesystem compression.
I'm not sure what mac has available.

As of now there are no plans to implement further compression support.
Closing and locking, as this is a duplicate of issue https://github.com/PCSX2/pcsx2/issues/8881/.
